Documentation/RunningTests.md

```diff
@@ -1,37 +1,73 @@
-## Running SerenityOS Tests
+# Running SerenityOS Tests
 
 There are two classes of tests built during a Serenity build: host tests and target tests. Host tests run on the build
 machine, and use [Lagom](../Meta/Lagom/ReadMe.md) to build Serenity userspace libraries for the host platform. Target
 tests run on the Serenity machine, either emulated or bare metal.
 
-### Running Host Tests
+## Running Host Tests
 
 There are two ways to build host tests: from a full build, or from a Lagom-only build. The only difference is the CMake
 command used to initialize the build directory.
 
 For a full build, pass `-DBUILD_LAGOM=ON` to the CMake command.
 
 ```sh
-mkdir Build
-cd Build
-cmake .. -GNinja -DBUILD_LAGOM=ON
+mkdir -p Build/i686
+cd Build/i8686
```

Comment:
Typo, but maybe we should use `Build/lagom` (like serenity.sh does) here anyway?

Reply:
Well, this first section is pointing out that you can build Serenity and Lagom in the same build directory. Would it make more sense to not even mention that part? Last time I used serenity.sh to build the Lagom target it built the whole system too 🤔

Reply:
`serenity.sh` uses the repository root rather than just the Lagom directory for its CMake invocation, however that doesn't mean it's building the whole system. Since the files end up in `Build/lagom` in that case (and are built for the host architecture, not `i686`), I think we should reflect that here.

Reply:
I must be using serenity.sh wrong because when I do `./Meta/serenity.sh test lagom` it builds 4k files instead of 540. I'll make the change though. Probably some changes to serenity.sh required to follow up and add an easy way test w/sanitizers.

Reply:
As mentioned on Discord, you were right about this - let's keep it like that for now and we can revisit when/if we update the script.